modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1091,6 +1092,8 @@ public ObjectEntry updateObjectEntry(
 				objectEntry.getObjectDefinitionId()),
 			objectEntryId, values);
 
+		objectEntryPersistence.clearCache(SetUtil.fromArray(objectEntryId));
+
```

Comment:
@marcelabc I'm not sure where we should place this.. here or after the `objectEntry = objectEntryPersistence.update(objectEntry);`, for example. Let's wait and see what Brian says.

Reply:
Hey @carolmariaabb, 
I tested your suggestion, but it did not work as expected.